modules/sandboxed-containers-scheduling-workloads.adoc

```diff
@@ -3,5 +3,60 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-scheduling-workloads_{context}"]
+= Scheduling {sandboxed-containers-first} workloads
 
-= Scheduling sandboxed containers workloads
+You can schedule your workloads to run on {sandboxed-containers-first}.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Add `runtimeClassName: kata` to any pod templated resources:
+* `Pod` objects
+* `ReplicaSet` objects
+* `ReplicationController` objects
+* `StatefulSet` objects
+* `Deployment` objects
+* `DeploymentConfig` objects
+
+.Example of `Pod` objects
```

Comment:
Would say either "Example of `Pod` object" or "Example for `Pod` objects". Also, I _think_ the guideline "Do not use backticks or other markup in assembly or module headings." applies to these example heading-type things as well.